swagger_server/itm/itm_scenario_session.py

```diff
@@ -894,13 +905,13 @@ def get_available_actions(self, session_id: str, scenario_id: str) -> List[Actio
 
         actions: List[Action] = []
         
-        # TODO When an action takes place, that action should be removed from the probeYaml's option list!
-        # tackle one probe at a time, after all available actions are returned for that probe remove it from remaining probes
         if self.current_isso.probe_system.probe_yamls:
             for option in self.current_isso.probe_system.probe_yamls[self.current_isso.probe_system.current_probe_index].options:
                 if (not self.kdma_training):
                     option.assoc_action.pop('kdma_association', None)
                 actions.append(option.assoc_action)
+            # Always allow tagging a casualty
+            actions.append(Action(action_id="tag_action", scenario_id=scenario_id, action_type='TAG_CASUALTY', unstructured="Tag a casualty"))
```

Comment:
I might be misreading the code but it seems like this is only adding one option for tagging a casualty. Shouldn't we have one available tag action per casualty? If so we can just iterate through the casualty list and add one tag action corresponding to those casualty id's.

Reply:
the for loop I believe is adding the other actions, the last one just makes sure tagging is always added, is how I read it.